proto/proxy/destination.proto

```diff
@@ -92,7 +92,7 @@ message TlsIdentity {
   // Verify the certificate based on the Kubernetes pod identity.
   message K8sPodIdentity {
     // The pod_identity string is of the format:
-    // {pod_name}.{pod_type}.{pod_ns}.conduit-managed.{controller_ns}.svc.cluster.local
+    // {owner_name}.{owner_type}.{pod_ns}.conduit-managed.{controller_ns}.svc.cluster.local
```

Comment:
This should also probably be updated to remove the `conduit-managed` fake service.

Reply:
Nevermind, I misunderstood what happened in #1221